.github/workflows/test.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
       - name: Run Go tests
-        run: go test -v ./...
+        run: find . -name go.mod -execdir go test -v ./... \;
```

Comment:
Unfortunately go test only runs in the context of the current module. This seems to be one of the accepted workarounds.